ironfish/src/telemetry/telemetry.ts

```diff
@@ -114,9 +117,35 @@ export class Telemetry {
   private metricsLoop(): void {
     Assert.isNotNull(this.metrics)
 
+    const inboundTrafficFields: Field[] = [
+      ...this.metrics.p2p_InboundTrafficByMessage.entries(),
+    ].map(([messageType, meter]) => {
+      return {
+        name: 'inbound_traffic_' + NetworkMessageType[messageType].toLowerCase(),
+        type: 'float',
+        value: meter.rate5m,
+      }
+    })
+
+    const outboundTrafficFields: Field[] = [
+      ...this.metrics.p2p_OutboundTrafficByMessage.entries(),
+    ].map(([messageType, meter]) => {
+      return {
+        name: 'outbound_traffic_' + NetworkMessageType[messageType].toLowerCase(),
+        type: 'float',
+        value: meter.rate5m,
+      }
+    })
+
     this.submit({
       measurement: 'node_stats',
       timestamp: new Date(),
+      tags: [
+        {
+          name: 'synced',
+          value: this.chain.synced.toString(),
+        },
+      ],
```

Comment:
Will this be ok for stats, now that were segmenting stats between synced and non synced nodes?

Reply:
It looks like we are using a `pivot` for [that query ](https://github.com/iron-fish/ironfish-telemetry/blob/3fc1b082f3f168ffe90881d28df2253c78e0965c/grafana/dashboards/dashboards.json#L475) so I think this should not affect the queries. It could affect performance but there's only 2 values so I feel like it should be relatively minor